File: sources/easygenerator.Web/App/mappers/organizationInviteMapper.js
import OrganizationInviteModel from 'models/organizations/organizationInvite';

export default {
map: (invite) => {
    return new OrganizationInviteModel({
        id: invite.Id,
        organizationId: invite.OrganizationId,
        organizationAdminFirstName: invite.OrganizationAdminFirstName,
        organizationAdminLastName: invite.OrganizationAdminLastName,
        organizationTitle: invite.OrganizationTitle
    });
}
}

File: sources/easygenerator.Web/Components/Mappers/Organizations/OrganizationInviteMapper.cs
using easygenerator.Infrastructure.DomainModel.Mappings;
using easygenerator.Web.Extensions;

namespace easygenerator.Web.Components.Mappers.Organizations
{
    public class OrganizationInviteMapper : IOrganizationInviteMapper
    {
        public dynamic Map(OrganizationInvite invite)
        {
            return new
            {
                Id = invite.Id.ToNString(),
                OrganizationId = invite.OrganizationId.ToNString(),
                OrganizationAdminFirstName = invite.OrganizationAdminFirstName,
                OrganizationAdminLastName = invite.OrganizationAdminLastName,
                OrganizationTitle = invite.OrganizationTitle
            };
        }
    }
}